DependencyInjection/Configuration.php

```diff
@@ -30,7 +27,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->booleanNode('autoinclude')->defaultTrue()->end()
-                ->scalarNode('base_path')->defaultValue('bundles/norzechowiczaceeditor/ace')->end()
+                ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
```

Comment:
I'm expecting people to get Ace the way they want :) They can use a CDN too...
The directory `vendor`is not created by composer, it it's in the `web` folder.

What is your recommendation on it?